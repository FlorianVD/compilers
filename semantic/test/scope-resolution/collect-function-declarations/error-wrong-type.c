// RUN-WITH-ARGS: -dump-function-table

foobar f() {}
