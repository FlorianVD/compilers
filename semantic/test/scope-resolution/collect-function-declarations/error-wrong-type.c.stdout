└── Program <2>
    └── FuncDecl: returnType = 'foobar', name = 'f' <1>
        └── CompoundStmt <0>
